inc/customizer/site-identity.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * Additional site identity customizer options.
+ *
+ * @class    Primer_Site_Identity_Options
+ * @package  Classes/Customizer
+ * @category Class
+ * @author   GoDaddy
+ * @since    NEXT
+ */
+
+class Primer_Site_Identity_Options {
+
+	/**
+	 * Class constructor.
+	 */
+	public function __construct() {
+
+		/**
+		 * Filter the site identity settings display.
+		 *
+		 * @since NEXT
+		 *
+		 * @var bool
+		 */
+		if ( ! (bool) apply_filters( 'primer_show_site_identity_settings', true ) ) {
+
+			return;
+
+		}
+
+		add_filter( 'primer_author_credit', array( $this, 'toggle_primer_author_credit' ) );
+
+		add_action( 'customize_register', array( $this, 'customize_register' ) );
+
+	}
+
+	/**
+	 * Toggle the visibility of the site credits in the footer.
+	 *
+	 * @filter primer_author_credit
+	 * @since  NEXT
+	 *
+	 * @return bool Returns true when `show_author_credit` theme mod is set.
+	 */
+	public function toggle_primer_author_credit() {
+
+		$show_author_credit = get_theme_mod( 'show_author_credit' );
+
+		return ! empty( $show_author_credit );
+
+	}
+
+	/**
+	 * Register additional site identity options.
+	 *
+	 * @action customize_register
+	 * @see    WP_Customize_Manager
+	 *
+	 * @since  NEXT
+	 *
+	 * @param  WP_Customize_Manager $wp_customize Instance of the WP_Customize_Manager class.
+	 */
+	public function customize_register( WP_Customize_Manager $wp_customize ) {
+
+		$wp_customize->add_setting(
+			'show_author_credit',
+			array(
+				'default'           => 1,
+				'sanitize_callback' => 'absint',
+			)
+		);
+
+		$wp_customize->add_control(
+			'page_width',
```

Comment:
@EvanHerman This looks like a typo, I think the control ID should be `show_author_credit`